src/app/creation/pages/creation-detail/creation-detail.component.ts

```diff
@@ -37,8 +37,8 @@ export class CreationDetailComponent implements OnInit {
     }
   }
 
-  public goToPortfolio = () => {
-    this.router.navigate(['/portfolio/creations']);
+  public goToCreations = () => {
```

Comment:
vu que tu utilises la méthode `navigate`, tu trouves pas que c'est plus logique de l'appeler comme ceci ?
```suggestion
  public navigateToCreations = () => {
```

Reply:
D'ailleurs en parlant de navigate, j'ai vu que certains utilisent navigateByUrl.
C'est quoi la différence exactement ?
Et quand utiliser l'un et quand l'autre, ou pourquoi ?

Reply:
J'ai pas vérifié mais je crois que le `navigateByUrl` c'est comme si tu accédais à l'url en la tapant dans la barre d'adresse, c'est à dire que ça te recharge tout.

Reply:
Ok.